Fix javadoc generation

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadoc) on project puzzler-core: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - /home/skapral/GitHub/puzzlerbot/puzzler-core/src/main/java/com/github/skapral/puzzler/core/text/threepars/TxtStandard.java:42: error: unknown tag: todo
[ERROR] * @todo #7 Split this object to two: one for splitting text to paragraphs,
```
...during `mvn clean install -Pextras`

--- a/pom.xml
+++ b/pom.xml
@@ -196,6 +196,15 @@
                                 </goals>
                             </execution>
                         </executions>
+                        <configuration>
+                            <tags>
+                                <tag>
+                                    <name>todo</name>
+                                    <placement>a</placement>
+                                    <head>To Do:</head>
+                                </tag>
+                            </tags>
+                        </configuration>
                     </plugin>
                 </plugins>
             </build>

--- a/puzzler-web/src/main/java/com/github/skapral/puzzler/github/operation/OpValidatingEventSignature.java
+++ b/puzzler-web/src/main/java/com/github/skapral/puzzler/github/operation/OpValidatingEventSignature.java
@@ -35,7 +35,7 @@ import java.util.function.Function;
 
 /**
  * Operation which is executed only if GitHub hook signatures
- * match on GitHub event. See <a href="https://developer.github.com/webhooks/securing/"/>
+ * match on GitHub event. See <a href="https://developer.github.com/webhooks/securing/"></a>
  * for details.
  *
  * @author Kapralov Sergey

